test/inductor/test_cpu_repro.py

```diff
@@ -353,6 +353,19 @@ def test_conv_transpose2d_packed_cpu(self):
                     (v,),
                 )
 
+    def test_scaled_dot_product_flash_attention_cpu(self):
```

Comment:
This affects gpu as well. Let's move this to `test_torchinductor.py` and enable for both cpu and gpu in `test_cpp_wrapper.py`.

Reply:
Moved to `test_torchinductor.py` and added it for both CPU and GPU in `test_cpp_wrapper.py`.